engines/real-time-ensembler/app.Dockerfile

```diff
@@ -0,0 +1,30 @@
+ARG BASE_IMAGE
+
+FROM ${BASE_IMAGE} as builder
+
+ARG MODEL_URL
+RUN gsutil -m cp -r ${MODEL_URL} .
+ARG FOLDER_NAME
+
+# Install dependencies required by the user-defined ensembler
+RUN /bin/bash -c ". activate ${CONDA_ENVIRONMENT} && conda env update --name ${CONDA_ENVIRONMENT} --file /${HOME}/${FOLDER_NAME}/conda.yaml"
+
+# Use conda-pack to create a standalone enviornment
+# in /venv:
+RUN conda-pack -n real-time-ensembler -o /tmp/env.tar && \
+  mkdir /venv && cd /venv && tar xf /tmp/env.tar && \
+  rm /tmp/env.tar
+
+RUN /venv/bin/conda-unpack
+
+FROM debian:latest
+
+COPY --from=builder /ensembler ./ensembler
+COPY --from=builder /pyfunc_ensembler_runner ./pyfunc_ensembler_runner
+COPY --from=builder /run.sh /run.sh
+COPY --from=builder /venv /venv
+
+RUN /bin/bash -c ". /venv/bin/activate && \
+    python -m pyfunc_ensembler_runner --mlflow_ensembler_dir /ensembler --dry_run"
```

Comment:
Oh it's just an option to run the web service (to load the ensembler from the mlflow registry and the runner) without actually serving it. It's... actually kinda pointless but I observed @gojek/merlin having it for their `pyfunc-server`s so I was wondering if that option would serve some other separate/greater purpose, which was why I included it to be safe. 

But I can remove it also since I can't seem to find any reason why we should run it once without serving it to ensure it's working, before actually running the service again for serving.

Reply:
> I was wondering if that option would serve some other separate/greater purpose, which was why I included it to be safe.

I think the reason is that if you run this command at the image building step, then failure of this `dry-run` step will prevent image from being even published and hence we will not spend more time trying to deploy the image if it's misconfigured

Reply:
Thanks for the clarification! Hmm but yet again in this particular instance, there isn't much difference between a `dry-run` and a 'normal-run' (i.e. we aren't calling additional methods or running any compute-heavy stuff), so there really isn't much opportunity to 'catch' an error early by performing the `dry-run` -- anything that fails in the dry run will also make the 'normal-run' fail, which stops the build process, and by extension any deployment. 

Maybe it'll just be a tiny bit more elegant without the `dry-run`?